projects/1.20/assets/snekcraft/snekcraft/lang/zh_cn.json

```
@@ -0,0 +1,64 @@
+{
+  "itemGroup.snekcraft": "蛇蛇工艺",
+  "creativetab.snake_tab": "蛇蛇工艺",
+
+  "item.snekcraft.snake_skin": "蛇蜕",
+  "item.snekcraft.snake_tooth": "蛇牙",
+  "item.snekcraft.snake_bag": "蛇袋",
+  "item.snekcraft.hognose_spawn_egg": "猪鼻蛇刷怪蛋",
+  "item.snekcraft.ball_python_spawn_egg": "球蟒刷怪蛋",
+  "item.snekcraft.terrarium_key": "容器钥匙",

```

```suggestion
  "item.snekcraft.terrarium_key": "玻璃箱钥匙",
```
这里主要是为统一Terrarium的称呼，检验后无误可不改

---

这个物品可以对wire_cage，aquarium，terrarium，lavaquarium四种容器使用

---

OK，不改